tests/utils/fixtures/fixtures.py

```diff
@@ -690,6 +690,8 @@ def test_foo():
     if mark is not None:
         features = mark.args
         current = bitbake_variables.get("MENDER_FEATURES", "").strip().split()
+        # For historical reasons we also check in DISTRO_FEATURES.
+        current += bitbake_variables.get("DISTRO_FEATURES", "").strip().split()
```

Comment:
I'm not going to require it, but it would be nice if we could use `min_yocto_version` above (probably needs to be refactored a bit then, since it is a fixture) to make a conditional choice here, to make sure we are not getting misleading results.

Reply:
Thanks, this could be applied to the other fixtures too.

I will merge this as-is for consistency with the rest, and may fix up what you describe the next time that I am dealing with old stable yocto branches :smiley: